changelog/unreleased/issue-2742

```diff
@@ -1,11 +1,10 @@
 Bugfix: Improve error handling for rclone and rest backend over HTTP2
 
-When retrieving data from the rclone / rest backend while also using HTTP2
+When retrieving data from the rclone / REST backend while also using HTTP2
 restic did not detect when no data was returned at all. This could cause
 for example the `check` command to report the following error:
-```
-Pack ID does not match, want xxxxxxxx, got e3b0c442
-```
+
+    Pack ID does not match, want [...], got e3b0c442
 
 This has been fixed by correctly detecting the incomplete download and
 retrying the download.
```

Comment:
Suggest "This has been fixed by correctly detecting and retrying the incomplete download".